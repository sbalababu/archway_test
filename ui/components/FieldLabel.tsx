import * as React from 'react';

interface Props {
  children: React.ReactNode;
}

const FieldLabel = ({ children }: Props) => (
  <div
    style={{
      color: 'rgba(0, 0, 0, .5)',
      textAlign: 'center',
      textTransform: 'uppercase',
      fontSize: 10,
    }}
  >
    {children}
  </div>
);

export default FieldLabel;
