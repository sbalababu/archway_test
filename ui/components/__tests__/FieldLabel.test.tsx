import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import FieldLabel from '../FieldLabel';

describe('FieldLabel', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<FieldLabel children={null} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
