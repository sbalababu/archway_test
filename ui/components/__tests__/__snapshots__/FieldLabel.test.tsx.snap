// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldLabel renders correctly 1`] = `
<div
  style={
    Object {
      "color": "rgba(0, 0, 0, .5)",
      "fontSize": 10,
      "textAlign": "center",
      "textTransform": "uppercase",
    }
  }
/>
`;
